arbor/include/arbor/mechcat.hpp

```diff
@@ -122,5 +122,6 @@ class mechanism_catalogue {
 
 const mechanism_catalogue& global_default_catalogue();
 const mechanism_catalogue& global_allen_catalogue();
+const mechanism_catalogue& global_bbp_catalogue();
```

Comment:
This is more of a TODO item than a request for change in this PR: Shouldn't we start to have a more dynamic catalogue of catalogues here? Something like a map<string, cat> and a register_catalogues helper? However
to make this right, we would need some kind of plugin mechanism.

Reply:
Let's bring it up on slack, maybe add an issue for it.

Reply:
I can see that it's already there's already an issue. Thanks!